src/main/java/frc/robot/subsystems/Intake2020.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+/*
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
+*/
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.drive.TalonController;
+
+public class Intake2020 extends SubsystemBase {
+
+    TalonController arm = new TalonController(Constants.INTAKE2020_ARM_MOTOR_ID);
+    TalonController roller = new TalonController(Constants.INTAKE2020_ROLLER_MOTOR_ID);
+    
+    Boolean up = true;
+
+    public Intake2020() {
+      super();
+    }
+
+    //stage 1
+
+      /**
+       * Lowers the arm so that it can pick up balls.
+      */
```

Comment:
Two comments here...
First, don't just comment as `//stage 1`. Instead write a more expressive comment, something like `// Stage 1 - Lower the grabber arm so we're ready to bring balls into the robot`.
Second, there's something wrong with the indents here. Please keep the indents consistent.